src/components/Single.jsx

```diff
@@ -157,10 +157,10 @@ class Single extends Component {
                             <b>Check your balance</b> by entering your BTCZ address on one of these explorers :
                         </p>
                         <ul style={{listStyleType: 'none'}}>
-                            <li><a href="https://explorer.btcz.rocks/">explorer.btcz.rocks</a></li>
+                            <li><a href="https://explorer.btcz.rocks/" target="_blank">https://explorer.btcz.rocks</a></li>
```

Comment:
In the footer, I got an error when I ran build because of missing ref tag or something. Can you run `yarn build` and ensure you don't see any errors? Thanks!

Reply:
no errors; added rel tag in new commit for target=_blanks
```sudo yarn build
yarn run v1.3.2
$ react-scripts build
Creating an optimized production build...
Compiled successfully.

File sizes after gzip:

  370.55 KB  build/static/js/main.1e0d753e.js
  1.9 KB     build/static/css/main.81599670.css

The project was built assuming it is hosted at https://paper.btcz.rocks.
You can control this with the homepage field in your package.json.

The build folder is ready to be deployed.
You may serve it with a static server:

  yarn global add serve
  serve -s build

Done in 153.85s.
```